Util/openl-maven-plugin/src/org/openl/rules/maven/VerifyMojo.java

```diff
@@ -182,7 +185,8 @@ String getHeader() {
         return "OPENL VERIFY";
     }
 
-    @SpringBootApplication
+    @SpringBootApplication(exclude = {DataSourceAutoConfiguration.class,
+            DataSourceTransactionManagerAutoConfiguration.class, HibernateJpaAutoConfiguration.class})

```

Comment:
Maybe it is better to use `@SpringBootConfiguration` with enabling Tomcat server?